toolbox/fdc3-conformance/Open-Tests.md

```diff
@@ -0,0 +1,66 @@
+
+
+## Wrong Context ![2.0](https://img.shields.io/badge/FDC3-2.0-blue)
+
+| App | Step            | Description                                                                                                                   |
+|-----|-----------------|-------------------------------------------------------------------------------------------------------------------------------|
+| A   | 1. Opening App     | A opens B using `fdc3.open(‘app B Name', ctx)`, but passes a context type that app B is NOT going to add a listener for (including NOT adding a listener for type `null`)                                                                |
+| B   | 2. Context not received | App B adds a context listener for a different type than the one A is sending.  <br/>Confirm that NO context is received.                         |
+| A   | 3. Promise         | Receives a rejection from the open promise with `AppTimeout` from <br>https://fdc3.finos.org/docs/api/ref/Errors#openerror |
+
+- `AOpensBWithWrongContext`: Perform above test
+
+## Multiple Listeners  ![1.2](https://img.shields.io/badge/FDC3-1.2-green) ![2.0](https://img.shields.io/badge/FDC3-2.0-blue)
+
+| App | Step            | Description                                                                                                                   |
+|-----|-----------------|-------------------------------------------------------------------------------------------------------------------------------|
+| A   | 1. Opening App     | A opens B using `fdc3.open(‘app B Name', ctx)`, sending an `fdc3.instrument` as context information    |
+| B   | 2. Context present | B opens and adds two context listeners:  first, a generic one, then second, a specific one for the `fdc3.instrument` context type.  <br/>Only the first listener will receive the `fdc3.instrument`    |
+
+-  `AOpensBMultipleListen`:  The correct (first) context listener should receive the context, and the promise resolves successfully in **A**.
+
+## Bad Context ![2.0](https://img.shields.io/badge/FDC3-2.0-blue)
+
+| App | Step            | Description                                                                                                                   |
+|-----|-----------------|-------------------------------------------------------------------------------------------------------------------------------|
+| A   | 1. Opening App     | A opens B using `fdc3.open(‘app B Name', ctx)`, but passes a context type that is semantically invalid (i.e. contains no `type` field)      |                                                          |
+| B   | 2. Context not received | App B adds an untyped context listener, but doesn't receive the context.        |
+| A   | 3. Promise         | Receives a rejection from the open promise with `AppTimeout` from <br>https://fdc3.finos.org/docs/api/ref/Errors#openerror |
+
+- `2.0-AOpensBNoListen`: Perform the above test. 
```

Comment:
As discussed at SWG meeting, malformed context handling is not yet defined in docs, so lets just drop this one for now and bring it back in 2.1
```suggestion
```